src/main/java/gov/cms/mat/cql_elm_translation/service/MadieFhirServices.java

```diff
@@ -0,0 +1,61 @@
+package gov.cms.mat.cql_elm_translation.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+import java.net.URI;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class MadieFhirServices {
+
+    private final RestTemplate restTemplate;
+
+    @Value("${madie.fhir.service.baseUrl}")
+    private String madieFhirService;
+
+    @Value("${madie.fhir.service.libraries.uri}")
+    private String librariesUri;
```

Comment:
Could this be a little more explicit?

Reply:
its a long name, but I updated it by adding hapi-fhir to it. Please take a look. Thank you